features/backend/product_attributes.feature

```diff
@@ -7,120 +7,73 @@ Feature: Product attributes
     Background:
         Given store has default configuration
           And there are following attributes:
-            | name               | presentation   |
-            | T-Shirt collection | Collection     |
-            | T-Shirt fabric     | T-Shirt fabric |
+            | name        | type     | code        | presentation |
+            | Book author | text     | book_author | Author       |
+            | Hardcover   | checkbox | hardcover   | Hardcover?   |
           And I am logged in as administrator
 
-    Scenario: Seeing index of all attributes
+    Scenario: Seeing attributes list
         Given I am on the dashboard page
-        When I follow "Configure attributes"
-        Then I should be on the product attribute index page
-        And I should see 2 attributes in the list
+         When I follow "Configure attributes"
```

Comment:
Aren't `Given/When/Then/And/Or` phrases supposed to be aligned to left, not right?